.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "plate-tectonics"]
+	path = plate-tectonics
+	url = https://github.com/Mindwerks/plate-tectonics.git
```

Comment:
I have not experience with submodules. The problem is that it makes sense to have plate-tectonics has a C library, independent by any python code. We want then to wrap it in a python module but to do that we need its code. Can we do in another way?